src/resources/Enums.ts

```diff
@@ -245,6 +245,25 @@ export enum SecurityProviderType {
     ZENDESK = 'ZENDESK',
 }
 
+export enum SecurityIdentityResult {
+    SUCCESS = 'Success',
+    ACCESS_DENIED = 'AccessDenied',
+    TIMED_OUT = 'TimedOut',
+    ENTITY_IS_INVALID = 'EntityIsInvalid',
+    ENTITY_IS_UNAVAILABLE = 'EntityIsUnavailable',
+    SECURITY_PROVIDER_IS_UNAVAILABLE = 'SecurityProviderIsUnavailable',
+    SECURITY_PROVIDER_IS_UNREACHABLE = 'SecurityProviderIsUnreachable',
+    SECURITY_PROVIDER_IS_NOT_READY = 'SecurityProviderIsNotReady',
+    UNEXPECTED_ERROR = 'UnexpectedError',
+}
+
+export enum SecurityIdentityType {
+    User = 'USER',
+    Group = 'GROUP',
+    VirtualGroup = 'VIRTUAL_GROUP',
+    Unknown = 'UNKNOWN',
+}
+
```

Comment:
I think we can reuse those https://github.com/coveo/platform-client/blob/master/src/resources/Enums.ts#L284-L302

Reply:
Ah, good catch. My bad! I'll fix that ASAP.

Reply:
Fixed